src/renderer/viz/expressions/aggregation/cluster/ClusterCount.js

```diff
@@ -1,11 +1,12 @@
 import BaseExpression from '../../base';
 import { checkMaxArguments } from '../../utils';
-import { CLUSTER_FEATURE_COUNT } from '../../../../schema';
+import { CLUSTER_FEATURE_COUNT } from '../../../../../constants/metadata';
 
 /**
  * Count of features per cluster.
  *
- * Note: `clusterCount` has no input parameters and if data is not aggregated, it always returns 1
+ * The `clusterCount` expression has no input parameters and if data is not aggregated, it always returns 1.
+ * Currently, it is not possible to use it as an input in global aggregations such as `globalMin` or `globalQuantiles`.
```

Comment:
I would probably remove 'Currently', as we're not really sure if we're going to make this soon